endpoints/openrtb2/amp_auction.go

```diff
@@ -232,7 +232,9 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 	}
 
 	// Populate any "missing" OpenRTB fields with info from other sources, (e.g. HTTP request headers).
-	deps.setFieldsImplicitly(r, reqWrapper, account)
+	if errs := deps.setFieldsImplicitly(r, reqWrapper, account); len(errs) > 0 {
+		errL = append(errL, errs...)
+	}
```

Comment:
Later after this call, In `getExtBidResponse`, errors of the warning type are added to the bid response. Debug warnings are ignored with the current approach.